File: .netWebApi/nemsport-web-api/Models/Member.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace nemsport_web_api.Models
{
    public class Member
    {
        // props
        public int Id { get; set; }
        public string Username { get; set; }
        public string Password { get; set; }
        public int MemberType { get; set; }

        // constructor
        public Member()
        { 
        }

        public Member(int id, string username, string password, int memberType) 
        {
            Id = id;
            Username = username;
            Password = password;
            MemberType = memberType;
        }

        public override string ToString()
        {
            return $"{nameof(Id)}: {Id}, {nameof(Username)}: {Username}, {nameof(Password)}: {Password}, {nameof(MemberType)}: {MemberType}";
        }
    }
}

File: app/src/main/java/com/dtu/nemsport/models/Member.kt
package com.dtu.nemsport.models

data class Member(val id:Int,
                  val username: String,
                  val password: String,
                  val memberType: String) {

    override fun toString(): String {
        return "id: $id - " +
                "username: $username - " +
                "memberType: $memberType"
    }
}
